pkg/platform/runtime/store/store.go

```diff
@@ -53,7 +52,7 @@ func New(installPath string) *Store {
 	}
 }
 
-func (s *Store) markerFile() string {
+func (s *Store) MarkerFile() string {
```

Comment:
I don't think this has to be exposed anymore, right? All marker file reading is now internal to the `store` package, yes?

Reply:
Yes, you're right. Thanks for catching that!